### ISSUE TITLE
we could have current news from various mars projects 

### ISSUE THREAD

**Issue by shirishag75** [mars-sim/mars-sim#89]
Maybe this is something we do post 1.0 release but it would be interesting if any interesting news or hypothesis about Mars comes in mainstream media, we could share the link which a gamer could click and read then or later on his default browser. 

It could be on any of the 16 roles we have depending on the ticks the gamer has made. Of course new findings, discoveries, inventions would change our modeling as well.

**Comment by mokun** [comment]
Yes. we can create a list of Breaking news from Earth, Moon colony, etc at the 2028 time frame.

We may have to create a category of subjects.

Let's list them out here.
